gnovm/cmd/gno/util.go

```diff
@@ -100,6 +100,60 @@ func gnoPackagesFromArgs(args []string) ([]string, error) {
 	return paths, nil
 }
 
+func gnoPackagePathsFromPattern(patterns []string) ([]string, error) {
```

Comment:
the logic here is wrong, you are walking dirs without `/...` pattern

Reply:
Can you please elaborate more about what's wrong here? I'm unable to figure out what's wrong here. I would be happy to correct :)